libs/sdk/src/wasm.ts

```diff
@@ -3,7 +3,8 @@ import wasm from './resources/sdk.wasm'
 
 export async function initWASM(): Promise<void> {
   try {
-    const go = new window.Go()
+    // @ts-expect-error globalThis.Go is defined
```

Comment:
`globalThis` works in both browser and non-browser environments. I could not figure out how to extend the type of the globalThis namespace, this silences the ts error.